f8a_report/helpers/report_helper.py

```diff
@@ -72,8 +72,13 @@ def cleanup_tables(self, table_name, column_name, num_days):
             sql.Identifier(column_name)
         )
         logger.debug('Starting to clean up "%s" table', table_name)
-        # Executing query
-        self.cursor.execute(query, (num_days,))
+        try:
+            # Executing query
+            self.cursor.execute(query, (num_days,))
+        except Exception as e:
+            logger.error("cleanup failed with exception %r", e)
+
+        # Commiting
         self.conn.commit()
```

Comment:
Are you sure commit won't generate any exception?

Reply:
Why dont you return from the function in the exception handling code?

Reply:
@arajkumar In case there is an error/exception and commit is inside try block it prevents the cleanup of other tables.That is what I observed.

Reply:
@prashbnair to keep commit() outside the exception handling block.